src/index.ts

```diff
@@ -25,6 +26,27 @@ export class KeyboardNavigation {
    */
   private originalTheme: Blockly.Theme;
 
+  /**
+   * Input mode tracking.
+   */
+  private inputModeTracker: InputModeTracker;
+
+  /**
+   * Focus ring in the workspace.
+   */
+  private workspaceFocusRing: Element | null = null;
+  /**
```

Comment:
```suggestion
  private workspaceFocusRing: Element | null = null;

  /**
```

Super nit: newline for consistency.